util/src/main/java/io/kubernetes/client/util/Yaml.java

```diff
@@ -358,6 +358,15 @@ public CustomRepresenter() {
       this.representers.put(IntOrString.class, new RepresentIntOrString());
       this.representers.put(byte[].class, new RepresentByteArray());
       this.representers.put(Quantity.class, new RepresentQuantity());
+      this.representers.put(DateTime.class, new RepresentDateTime());
+    }
+
+    private class RepresentDateTime implements Represent {
+      @Override
+      public Node representData(Object data) {
+        DateTime dateTime = (DateTime) data;
```

Comment:
Why is dateTime cast from data when then not used in next line for conversion to String?

Reply:
Removed.